cypress/integration/visitor/loginPage/signUp.spec.js

```diff
@@ -6,14 +6,14 @@ describe('Sign Up', () => {
   });
 
   it('Visitor singns up with valid credentials', () => {
-    const timestemp = +new Date();
+    const timestamp = +new Date();
     const { firstName, lastName } = this.user;
 
     const validCredentials = {
       firstName,
       lastName,
-      email: `${firstName}-${timestemp}@gmail.com`,
-      password: timestemp,
+      email: `${firstName}-${timestamp}@gmail.com`,
```

Comment:
we should use domain for tests and examples  
https://tools.ietf.org/html/rfc2606